app/services/induction/amend_cohort_after_eligibility_checks.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+# Change the cohort of an ECF participant whose eligibility has been recalculated.
+# If the participant has become eligible and is currently in a payments-frozen cohort,
+# transfer them to the currently active registration cohort.
+module Induction
+  class AmendCohortAfterEligibilityChecks
+    include ActiveModel::Model
+
+    attr_accessor :participant_profile
+
+    def call
+      amend_participant_cohort if eligible_participant_in_frozen_cohort?
+    end
+
+  private
+
+    def amend_participant_cohort
+      Induction::AmendParticipantCohort.new(participant_profile:,
```

Comment:
`save!` since we're not checking the return value?

Reply:
Well... the code calling this SO actually checks the value.